### PR TITLE
Bump spanner lib to v1.5.0

### PR DESCRIPTION
**Pull request by dmivankov**
Will enable spanner emulator support
https://cloud.google.com/spanner/docs/emulator#go

Current version supports SPANNER_EMULATOR_HOST but fails at run-time
with spanner permission errors.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,9 +1,8 @@
 module github.com/golang-migrate/migrate/v4
 
 require (
-	cloud.google.com/go v0.53.0 // indirect
-	cloud.google.com/go/spanner v1.2.0
-	cloud.google.com/go/storage v1.5.0
+	cloud.google.com/go/spanner v1.5.0
```
Comment (dmivankov) [inline go.mod:4]: Only this line was changed manually, other changes are from running "go get"

Comment (dhui) [reply go.mod:4]: Could you also run `go mod tidy`?

Comment (dmivankov) [reply go.mod:4]: Done, though still a big diff

**Comment by wyardley** [comment]
Thanks for adding this, btw... was in the middle of checking if the program supported the emulator and found this.

**Pull request merged**